src/init.cpp

```diff
@@ -71,6 +75,10 @@ bool fFeeEstimatesInitialized = false;
 static CZMQNotificationInterface* pzmqNotificationInterface = NULL;
 #endif
 
+#if ENABLE_PROTON
+static AMQPNotificationInterface* pAMQPNotificationInterface = NULL;
```

Comment:
Why ``CZMQNotificationInterface``, but ``AMQPNotificationInterface`` without the ``C``?